setup.cfg

```diff
@@ -33,7 +33,7 @@ apache_airflow_provider =
 
 [options.extras_require]
 dev =
-    apache-airflow
+    apache-airflow>=2.2.0
```

Comment:
Should we add the same line to the `install_requires` section as well?

Reply:
Good point, thanks